pkg/controller/node/controller_utils.go

```diff
@@ -40,45 +40,37 @@ const (
 	LargeClusterThreshold = 20
 )
 
-// deletePods will delete all pods from master running on given node, and return true
-// if any pods were deleted, or were found pending deletion.
-func deletePods(kubeClient clientset.Interface, recorder record.EventRecorder, nodeName, nodeUID string, daemonStore cache.StoreToDaemonSetLister) (bool, error) {
-	remaining := false
+func getPodsOfNode(kubeClient clientset.Interface, nodeName string) (*api.PodList, error) {
```

Comment:
I'll be surprised if we wouldn't have such a function already. @lavalamp @caesarxuchao @deads2k - do you know if we do?